### DISCUSSION TITLE
Add support to donations

### DISCUSSION THREAD

**Discussion by dzmitry-lahoda** [nix-community/ethereum.nix#324] [Ideas]
please add if possible

**Comment by selfuryon** [comment]
What do you think about that, @aldoborrero @brianmcgee? Or it has some legal issues?

I see that "Open Collective" supports [crypto donation](https://docs.opencollective.com/help/financial-contributors/crypto) in beta.

At least, nix-community admins or we can create a multi-sig gnosis safe wallet for donation?!

**Comment by aldoborrero** [comment]
I'm considering multiple options, we have the following ones:

- [ ] [Github Sponsors](https://github.com/sponsors)
- [ ] [Gitcoin Grants](https://bounties.gitcoin.co/grants)
- [ ] [Ethereum Foundation Grants](https://ethereum.org/en/community/grants/)

In terms of donations we can use the following options:

- [ ] Using a multi-sig Gnosis Safe Wallet (as you described).
- [ ] Using the Crypto Donation button from "Open Collective" (as you described).

Let me know if we're missing more alternatives to consider first. It will certainly have legal implications, so we need to research each option accordingly.

**Reply by selfuryon** [reply]
I also know about [SSV Grants](https://grants.ssv.network/) and [RP Grants](https://dao.rocketpool.net/c/grant-bounties/13). Anyway, grants are a very interesting topic, should we try to get one?

**Reply by aldoborrero** [reply]
Indeed. But first, before doing so I would like to finish the refactoring/improvements to the documentation website I'm implementing currently. 

Tomorrow I'll flesh out the roadmap I would like to see for the foreseeable future of the project.

**Comment by aldoborrero** [comment]
I think this should be moved to a "discussion" as it fits more there.
